ui/imports/shared/popups/send/panels/RouterPanel.qml

```diff
@@ -0,0 +1,196 @@
+﻿import QtQuick 2.13
```

Comment:
Please add storybook page. Otherwise it's hard to properly review that component and inspect API.